classes/ActionScheduler_WPCommentCleaner.php

```diff
@@ -81,7 +85,7 @@ public static function maybe_schedule_cleanup() {
 	public static function delete_all_action_comments() {
 		global $wpdb;
 		$wpdb->delete( $wpdb->comments, array( 'comment_type' => ActionScheduler_wpCommentLogger::TYPE, 'comment_agent' => ActionScheduler_wpCommentLogger::AGENT ) );
-		delete_option( self::$has_logs_option_key );
+		update_option( self::$has_logs_option_key, 'no' );
```

Comment:
```suggestion
		update_option( self::$has_logs_option_key, 'no', true );
```

Same as above comment.